Misc/NEWS.d/next/Core and Builtins/2022-05-20-13-32-24.gh-issue-15870.e9B-pv.rst

```diff
@@ -0,0 +1,4 @@
+Added a new stable API function ``PyType_FromMetaModuleAndSpec``, which
+mirrors the behavior of ``PyType_FromModuleAndSpec`` except that it takes an
+additional metaclass argument. This is, e.g., useful for language binding
+tools that need to store additional information in the type object.
```

Comment:
Suggesting to add links to the C API docs, drop the 'e.g.' abbreviation, and most important, mention that this function is exposed in the Limited API.

```suggestion
Added the new function :c:func:`PyType_FromMetaModuleAndSpec`, which mirrors
the behavior of :c:func:`PyType_FromModuleAndSpec`, except that it takes an
additional metaclass argument. This is useful for language binding tools that
need to store additional information in the type object.

``PyType_FromMetaModuleAndSpec`` is available in the Limited API.
```

Reply:
Done, thanks. I added a bit more text explaining two use cases that were relevant in the case of nanobind, to provide further context for this change.